package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "test": "./node_modules/jasmine/bin/jasmine.js",
-    "build": "mkdir -p dist && ./node_modules/browserify/bin/cmd.js ./src/imgix.js > ./dist/imgix.js && ./node_modules/uglify-js/bin/uglifyjs ./dist/imgix.js --mangle > ./dist/imgix.min.js"
+		"build": "mkdir -p dist && ./node_modules/browserify/bin/cmd.js ./src/imgix.js > ./dist/imgix.js && ./node_modules/uglify-js/bin/uglifyjs ./dist/imgix.js --mangle > ./dist/imgix.min.js",
```

Comment:
Lil tab/space war goin on here, huh?

Reply:
A bit! Prettier has different ideas to NPM 🤔

Reply:
We should lock em in a room and have em fight it out

Reply:
I guess yarn will just sit in the corner and eat popcorn